src/main/java/org/folio/services/BatchGroupService.java

```diff
@@ -48,8 +48,7 @@ public Future<BatchGroup> createBatchGroup(BatchGroup batchGroup, RequestContext
   public Future<BatchGroupCollection> getBatchGroups(int limit, int offset, String query, RequestContext requestContext) {
     String queryParam = getEndpointWithQuery(query);
     String endpoint = String.format(GET_BATCH_GROUPS_BY_QUERY, limit, offset, queryParam);
-    return restClient.get(endpoint, BatchGroupCollection.class, requestContext)
-      .onFailure(t -> logger.error("Error getting batch groups", t));
+    return restClient.get(endpoint, BatchGroupCollection.class, requestContext);
```

Comment:
it will be great, if there are add more logs (if possible)

Reply:
Answered in the PM: RestClient always logs errors so this one was duplicate

Reply:
thanks